docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,543 @@
+* `Confluent REST Proxy <https://docs.confluent.io/current/kafka-rest/docs/api.html#post--topics-(string-topic_name)-partitions-(int-partition_id)>`_
+* `Non-Java clients based on librdkafka <https://docs.confluent.io/current/clients/index.html>`_ , including Confluent Python, Confluent Go, Confluent DotNet
+
+
+Centralized Schema Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Schemas in Schema Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At this point, you have producers serializing Avro data and consumers deserializing Avro data.
+The producers are registering schemas and consumers are retrieving schemas.
+You can view subjects and associated schemas via the REST endpoint in |sr|.
+
+View all the subjects registered in |sr| (assuming |sr| is running on the local machine listening on port 8081):
+
+.. sourcecode:: bash
+
+   $ curl --silent -X GET http://localhost:8081/subjects/ | jq .  
+   [
+     "transactions-value"
+   ]
+
+In this example, the Kafka topic `transactions` has messages whose value, i.e., payload, is Avro.
+View the associated subject `transactions-value` in |sr|:
+
+.. sourcecode:: bash
+
+   $ curl --silent -X GET http://localhost:8081/subjects/transactions-value/versions/latest | jq .
+   {
+     "subject": "transactions-value",
+     "version": 1,
+     "id": 1,
+     "schema": "{\"type\":\"record\",\"name\":\"Payment\",\"namespace\":\"io.confluent.examples.clients.basicavro\",\"fields\":[{\"name\":\"id\",\"type\":\"string\"},{\"name\":\"amount\",\"type\":\"double\"}]}"
+   }
+
+Let's break down what this version of the schema defines
+
+* `subject`: the scope in which schemas for the messages in the topic `transactions` can evolve
+* `version`: the schema version for this subject, which starts at 1 for each subject
+* `id`: the globally unique schema version id, unique across all schemas in all subjects
+* `schema`: the structure that defines the schema format
+
+The schema is identical to the :ref:`schema file defined for Java client applications<schema_registry_tutorial_definition>`.
+Notice in the output above, the schema is escaped JSON, i.e., the double quotes are preceded with backslashes.
+
+Based on the schema id, you can also retrieve the associated schema by querying |sr| REST endpoint:
+
+.. sourcecode:: bash
+
+   $ curl --silent -X GET http://localhost:8081/schemas/ids/1 | jq .
+   {
+     "schema": "{\"type\":\"record\",\"name\":\"Payment\",\"namespace\":\"io.confluent.examples.clients.basicavro\",\"fields\":[{\"name\":\"id\",\"type\":\"string\"},{\"name\":\"amount\",\"type\":\"double\"}]}"
+   }
+
+If you are using |c3|, you can view the topic schema easily from the UI:
+
+.. figure:: c3-schema-transactions.png
+    :align: center
+
+
+
+Schema IDs in Messages
+^^^^^^^^^^^^^^^^^^^^^^
+
+Integration with |sr-long| means that Kafka messages do not need to be written with the entire Avro schema.
+Instead, Kafka messages are written with the schema _id_.
```

Comment:
The underscores here were taken literally on the staging server. Not sure if that should actually be italiziced or bold?